app/models/cassandra/classification.rb

```diff
@@ -0,0 +1,59 @@
+class Cassandra::Classification
+  include Cequel::Record
+
+  key :project_id, :int
+  key :workflow_id, :int
+  key :subject_id, :int
+  key :workflow_version, :int
+  key :user_ip, :inet
+  key :user_id, :int
+  column :created_at, :timestamp
+  column :updated_at, :timestamp
+  column :user_group_id, :int
+  column :completed, :boolean
+  column :gold_standard, :boolean
+  column :expert_classifier, :boolean
+  column :metadata, :text
+  column :annotations, :text
+
+  after_create :save_subject
+
+  def self.from_ar_model(classification)
+    attrs = classification.attributes.dup
+    attrs.delete("id")
```

Comment:
@ggdhines will including the panoptes id key break things or will the aggregation engine just not run for projects like stormwatch? We're only interested in the former, the later can be punted till we run a project like this.